lms/djangoapps/certificates/tests/test_webview_views.py

```diff
@@ -20,12 +20,6 @@
 from common.djangoapps.student.tests.factories import CourseEnrollmentFactory, UserFactory
 from common.djangoapps.track.tests import EventTrackingTestCase
 from common.djangoapps.util.date_utils import strftime_localized
-from lms.djangoapps.badges.events.course_complete import get_completion_badge
-from lms.djangoapps.badges.tests.factories import (
-    BadgeAssertionFactory,
-    BadgeClassFactory,
-    CourseCompleteImageConfigurationFactory
-)
 from lms.djangoapps.certificates.config import AUTO_CERTIFICATE_GENERATION
 from lms.djangoapps.certificates.models import (
     CertificateGenerationCourseSetting,
```

Comment:
Shouldn't we remove following feature flags as welll?
https://github.com/openedx/edx-platform/blob/cab920e0af877848a96643940c6a933e2546af31/lms/djangoapps/certificates/tests/test_webview_views.py#L55